### PR TITLE
Create labeler

### PR DESCRIPTION
**Pull request by pheiduck**
Do I missed some labels we can automate?

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- any: ['runtime/doc/if_mzsch.txt']
+
+if_perl:
+- any: ['src/if_perl.xs']
+- any: ['src/if_perlsfio.c']
+- any: ['src/proto/if_perl.pro']
+- any: ['src/proto/if_perlsfio.pro']
+- any: ['runtime/doc/if_perl.txt']
+
+if_python:
```
Comment (zeertzjq) [inline .github/labeler.yml:29]: `if_py_both.h`

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (zeertzjq) [inline .github/labeler.yml:29]: `if_pyth.txt`

**Comment by pheiduck** [comment]
@chrisbra what do you think about this PR?

**Comment by chrisbra** [comment]
should be good I think. Perhaps add a label for the general runtime files, like `runtime/ftplugin`, `runtime/syntax`, `runtime/indent`? and perhaps one for termdebug plugin.

**Comment by pheiduck** [comment]
@chrisbra it's updated

**Comment by yegappan** [comment]
Can you also add the label "plugin-netrw" for the Netrw plugin related files?

**Review by yegappan — APPROVED** [review]

**Comment by chrisbra** [comment]
I added a few more labels. Thanks!

**Pull request closed without merging**